docker/README.md

```diff
@@ -1,10 +1,12 @@
-# NVIDIA Merlin Dockerfiles
+# NVIDIA Merlin Docker Containers
```

Comment:
This repo also contains the dockerfiles used to build the containers - and I think we should mention that